TypeCobol.Test/Report/ReportTestHelper.cs

```diff
@@ -73,8 +74,17 @@ public static void ParseWithNodeListenerReportCompare(string fileName, string re
                             string result = sw.ToString();
                             string expected = File.ReadAllText(output, format.Encoding);
                             TypeCobol.Test.TestUtils.compareLines(input, result, expected, PlatformUtils.GetPathForProjectFile(output));
+                            return true;
                         }
                     }
+                    else
+                    {
+                        return false;
+                    }
+                }
+                else
+                {
+                    return false;
```

Comment:
The two blocks `else { return false; }` can be simplified as `return false;`at the end of the try block.

Reply:
Just a matter of style